coral/src/app/pages/Topics.tsx

```diff
@@ -0,0 +1,20 @@
+import Layout from "src/app/layout/Layout";
+import { BrowseTopics } from "src/app/features/topics/BrowseTopics";
+import { Typography } from "@aivenio/design-system";
+
+const Topics = () => {
+  return (
+    <Layout>
+      <>
+        <Typography.Heading htmlTag={"h1"}>
+          Browse all topics
+        </Typography.Heading>
+        <div style={{ paddingTop: "30px" }}>
```

Comment:
For sake of consistency, we could wrap the header and list into a DS `<Grid` or `<Flexbox` and use a `row-gap` value from DS.

Reply:
Sounds good!